src/main/kotlin/com/jetbrains/snakecharm/inspections/quickfix/CreateEnvFile.kt

```diff
@@ -0,0 +1,50 @@
+package com.jetbrains.snakecharm.inspections.quickfix
+
+import com.intellij.codeInspection.LocalQuickFixOnPsiElement
+import com.intellij.openapi.project.Project
+import com.intellij.openapi.vfs.InvalidVirtualFileAccessException
+import com.intellij.psi.PsiDocumentManager
+import com.intellij.psi.PsiElement
+import com.intellij.psi.PsiFile
+import com.intellij.psi.PsiManager
+import com.jetbrains.snakecharm.SmkNotifier
+import com.jetbrains.snakecharm.SnakemakeBundle
+import java.io.IOException
+
+class CreateEnvFile(expr: PsiElement, private val fileName: String) : LocalQuickFixOnPsiElement(expr) {
+    private val defaultContext = """
+        channels:
+        dependencies:
+    """.trimIndent()
+
+    override fun getFamilyName() = SnakemakeBundle.message("INSP.NAME.conda.env.missing.fix", fileName)
+
+    override fun getText() = familyName
+
+    override fun invoke(project: Project, file: PsiFile, startElement: PsiElement, endElement: PsiElement) {
+        val tokens = fileName.split('/')
+        val targetName = tokens.lastOrNull() ?: return
+        var currentFile = file.virtualFile.parent
+        try {
+            tokens.forEach {
```

Comment:
It is reasonable to expect, that in such complicated application, like IntelliJ platform the task of creating file by relative path + create intermediate dirs isn't a rare case and already solved via some API. The code below is bicycle invention in a rather complected way. E.g. java.io already have methods for creating files and folders. So here we could use smth like: 

```kotlin
        val targetFile = file.virtualFile.parent.findFileByRelativePath(fileName)
        // if targetFile == null ....
        val targetFileFolder = targetFile.parent
        VfsUtil.createDirectoryIfMissing(targetFileFolder.path)
        LocalFileSystem.getInstance().createChildFile(this, targetFileFolder, targetFile.name)
```

or just:
```kotlin
        val targetFile = Paths.get(file.virtualFile.parent.path, fileName)
        Files.createDirectories(targetFile.parent)
        Files.createFile(targetFile)
```